src/optim_tests/multivariate/MultivariateProblems.jl

```diff
@@ -1,6 +1,6 @@
 module MultivariateProblems
 
-import Base.gradient
+import LinearAlgebra: gradient, dot
```

Comment:
I don't think we need to import gradient anymore :)

Reply:
Ah, right! I was just going by the warnings, but that cannot know that we actually didn't want to extend LinearAlgebra's/Base's gradient.